cogs/utils/db.py

```diff
@@ -172,6 +172,21 @@ async def set_member(guildID, memberID, option, value):
     await conn.close()
 
 
+# Returns members who have birthday on a given date
+async def get_members_bday_equals_date(date):
+    conn = await db_connect()
+    c = await conn.cursor()
+    sql = f"""
+    SELECT MIN(member_id) AS member_id, *
```

Comment:
What does `SELECT MIN(member_id) AS member_id, *` accomplish here that `SELECT member_id` would not?

Reply:
To avoid duplicates in the result

Reply:
Doesn't `MIN()` simply return the smallest value in a list of values? Would it not be better to use `DISTINCT` to filter out duplicate rows?